multiqc/modules/dragen/ploidy_estimation_metrics.py

```diff
@@ -16,10 +16,12 @@
 
 class DragenPloidyEstimationMetrics(BaseMultiqcModule):
     def add_ploidy_estimation_metrics(self):
-        data_by_sample = dict()
+        data_by_sample = {}
+        headers = defaultdict(dict)
 
         for f in self.find_log_files("dragen/ploidy_estimation_metrics"):
             s_name, data = parse_ploidy_estimation_metrics_file(f)
+            for d in data: headers[d]['title'] = d
```

Comment:
Has this code already been tested? IMHO this line should not work in python, but did not test this myself yet

Reply:
I've tested it with several CPython versions. Shall work fine for 3.6 and above. The line can be always split if other Python implementations could have any problem with that.